cmd/openshift-install/migrate.go

```diff
@@ -0,0 +1,186 @@
+package main
```

Comment:
i would like the structure to be

```
migrate azure privatedns <eligible | migrate>
```

Reply:
and hopefully, the package structure looks like
```
openshift-install/
---- migrate
-------- eligible.go
-------- migrate.go
```

Reply:
I've updated the directory structure to this. I've added an extra azure directory where the command files are under. If you don't like that I can remove it and put it this way.